examples/helpers/ios/ViewController.mm

```diff
@@ -42,9 +42,11 @@ - (void)viewDidLoad {
 - (void)runExampleProgram {
   // Run example in background thread to not block the UI
   __weak ViewController* weakSelf = self;
+  const AccessKey access_key{"", ""};  // put your here.access.key.id and here.access.key.secret
```

Comment:
Just a comment - we definitely need to improve our iOS examples with some text fields, so, user can enter credentials from UI. but we can do it later.